brave/brave-ios
# Fix 2436: Implement PaymentResponse

## Summary of Changes

This pull request fixes #2436 

## Submitter Checklist:


## Test Plan:
1. Navigate to https://test-sku-store.bravesoftware.com/sku-dev/demo_cart and complete the payment workflow. Verify the Status is displayed after the button
2. Navigate to https://test-sku-store.bravesoftware.com/sku-dev/demo_one_click and complete the payment workflow. Verify the Status is displayed after the button

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
